android/java/res/layout/bottom_toolbar_browsing.xml

```diff
@@ -51,7 +51,7 @@
         android:paddingTop="@dimen/search_accelerator_height_padding"
         android:paddingBottom="@dimen/search_accelerator_height_padding"
         android:contentDescription="@string/accessibility_toolbar_btn_search_accelerator"
-        android:src="@drawable/ic_search"/>
+        android:src="@drawable/ic_search_21dp"/>
```

Comment:
We should use app:srcCompat with vector drawables.

Reply:
Fixed.